explorer/explorertypes.go

```diff
@@ -510,10 +510,10 @@ type AddrPrefix struct {
 // AddressPrefixes generates an array AddrPrefix by using chaincfg.Params
 func AddressPrefixes(params *chaincfg.Params) []AddrPrefix {
 	Descriptions := []string{"P2PK address",
-		"P2PKH address prefix",
-		"P2PKH address prefix",
+		"P2PKH address prefix. Standard wallet address. 1 public key -> 1 private key",
+		"Ed25519 high-speed signature",
```

Comment:
Still should say it's a P2PKH address (just using the edwards sig algo), similar to the description "secp256k1 Schnorr P2PKH address prefix"